### PR TITLE
Issue 275

### PR DESCRIPTION
**Pull request by SetCodesToFire**
Solving  #275

### CODE REVIEW

**Comment by SetCodesToFire** [comment]
Hi @chrisfilo  ,
Codeclimate's check need to be resolved for this PR

File: tests/tsv.spec.js [hunk 1]
```diff
@@ -109,6 +109,21 @@ describe('TSV', function(){
...
+    };
+
+    it("should not allow _scans.tsv files without filename column", function () {
+        var tsv = 'header-one\theader-two\theader-three\n' +
+                  'value-one\tvalue-two\tvalue-three';
+        validate.TSV.TSV(scansFile, tsv, [], function (issues) {
+            assert(issues.length === 1 && issues[0].code === 68);
+        });
+    });
+
```
Comment (chrisgorgo) [inline tests/tsv.spec.js:126]: Could you add a test where filename column is present and there are no errors?

**Review by chrisgorgo — CHANGES_REQUESTED** [review]

File: tests/tsv.spec.js (hunk 1 above)
Comment (SetCodesToFire) [reply tests/tsv.spec.js:126]: okay sure.

**Comment by chrisgorgo** [comment]
Awesome! Thanks!

**Pull request merged**